source/includes/plugin_infos/_040-index.md.erb

```diff
@@ -0,0 +1,62 @@
+## Get all plugin_infos
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/plugin_infos' \
+      -u 'username:password' \
+      -H 'Accept: application/vnd.go.cd.v1+json'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.go.cd.v1+json; charset=utf-8
+ETag: "3924a894cf0e5bef02abe9de0df3bb84"
+```
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "https://ci.example.com/go/api/admin/plugin_infos"
+    },
+    "doc": {
+      "href": "http://api.go.cd/#plugin-infos"
+    }
+  },
+  "_embedded": {
+    "plugin_infos": [
+      {
+        "_links": {
+          "self": {
+            "href": "https://ci.example.com/go/api/admin/plugin_infos/plugin_id"
+          },
+          "doc": {
+            "href": "http://api.go.cd/#plugin-infos"
+          },
+          "find": {
+            "href": "https://ci.example.com/go/api/admin/plugin_infos/:id"
+          }
+        },
+        "id": "plugin_id",
+        "name": "SCM Plugin",
+        "display_name": "SCM Plugin For HG",
+        "version": "1.2.3",
+        "type": "scm"
+      }
+    ]
+  }
+}
+```
+
+Lists all available plugin infos.
+
+<%= available_since('16.7.0') %>
+
+<p class='http-request-heading'>HTTP Request</p>
+
+`GET /go/api/admin/plugin_infos`
```

Comment:
I'm wondering if we should call this `plugins_info` instead — `plugin_infos` seems gramatically incorrect, since informations is not a word, and the api returns information about plugins?